Importer/Log/CsvLog.php

```diff
@@ -81,19 +80,19 @@ public function getCriticalErrors()
     }
 
     /**
-     * @return bool
+     * {@inheritdoc}
      */
     public function containErrors()
     {
         return count($this->getConfigurationErrors()) + count($this->getDataErrors()) + count($this->getCriticalErrors()) > 0;
     }
 
     /**
-     * @param string $csv
+     * @param array $rawData
      */
-    public function setCsv($csv)
+    public function setRawData(array $rawData)
```

Comment:
if this is for CSVs only, then we can call it rawCsvLines or something (to make sure we know what it is)